bats/foreign-keys.bats

```diff
@@ -0,0 +1,1236 @@
+#!/usr/bin/env bats

```

Comment:
Some newlines in between chunks of setup and test assertions would make these tests easier to read, especially when you have multiple dolt sql <<SQL heredoc chunks